.travis.yml

```diff
@@ -118,6 +118,11 @@ jobs:
       env: >-
         FILE_ENV="./ci/test/00_setup_env_i686.sh"
 
+    - stage: test
+      name: 'x86_64 Linux  [GOAL: install]  [CentOS 7]  [no depends, only system libs]'
+      env: >-
+        FILE_ENV="./ci/test/00_setup_env_native_centos.sh"
```

Comment:
Our travis is currently running at its limit and this doesn't seem like it is adding a lot new coverage compared to the other builds. What about only running it in cirrus instead of travis?

Reply:
Could CentOS build be a replacement for Trusty build on Travis?

Reply:
Sounds good

Reply:
Should Trusty build be removed from Travis in this PR?

Reply:
Yeah, I think this has been the plan: https://github.com/bitcoin/bitcoin/issues/17628#issuecomment-559448201

Reply:
@emilengler's 5423bd3474581a479b244cb9b599cbefafc64618 has been cherry-picked.